pom.xml

```diff
@@ -31,14 +31,35 @@
         <dependency>
             <groupId>com.cloudbees</groupId>
             <artifactId>syslog-java-client</artifactId>
-            <version>1.1.2</version>
+            <version>1.1.7</version>
         </dependency>
         <dependency>
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>jsr305</artifactId>
             <version>3.0.2</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins</groupId>
+            <artifactId>configuration-as-code</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.jenkins</groupId>
+            <artifactId>configuration-as-code</artifactId>
+            <classifier>tests</classifier>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.4</version>
+        </dependency>
```

Comment:
Please use https://github.com/jenkinsci/apache-httpcomponents-client-4-api-plugin instead to avoid the necessity to make a security update everytime the lib provides a patch.

Reply:
Thanks for the tip.  I have made that change now